agent/tool-scripts/pprof

```diff
@@ -116,12 +122,12 @@ get_masters() {
     while true; do
         i=1
         for master in "${masters[@]}"; do
-            oc get --raw /debug/pprof/profile?seconds=${interval} --server https://${master}:8443 > ${master}_api_profile && ${tool_bin} tool ${tool} -text -seconds=${interval} -output $PPROF_TMPDIR/${master}/$(date +%Y-%m-%d:%H:%M:%S)_api_profile.txt /tmp/${master}_api_profile &
+            oc get --raw /debug/pprof/profile?seconds=${interval} --server https://${master}:8443 > /tmp/${master}_api_profile && ${tool_bin} tool ${tool} -text -seconds=${interval} -output $PPROF_TMPDIR/${master}/$(date +%Y-%m-%d:%H:%M:%S)_api_profile.txt /tmp/${master}_api_profile &
```

Comment:
Can we consider doing one of the two choices below?

 * change references to `/tmp` to be `$dir`, changing line 21 as well to set `dir` to be the value of `pbench_tmp`
 * remove the use of `$dir`, and the variable definition at line 21, and use `$pbench_tmp` everywhere?

Reply:
Where is `${pbench_tmp}` declared?

Reply:
Is provided by line 8.